### PR TITLE
Panel high contrast border

### LINKED ISSUE #8588: In high contrast, Panel isn't distinguishable from the background
Hi folks,

When in high contrast, it appears that the Panel isn't distinguishable from the main content.

Expected:
image
(image taken from https://github.com/OfficeDev/office-ui-fabric-react/pull/7155)

Actual:
image

As you can see, the border of the Panel isn't visible. The regression came in [6.141.0](https://github.com/OfficeDev/office-ui-fabric-react/releases/tag/office-ui-fabric-react_v6.141.0), though I haven't investigate any further. 

Thanks :)

### PR DESCRIPTION
**Pull request by kkjeer**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #8588 
- [x] Include a change request file using `$ npm run change`

#### Description of changes

This pull request adds a left and right border to the main Panel div in high contrast mode.

Before (Panel indistinguishable from the rest of the page):
panel-hc-before

After: (Panel separated from the rest of the page with the border):
panel-hc-after

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/8609)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts [hunk 1]
```diff
@@ -258,6 +258,11 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
           ['@supports (-webkit-overflow-scrolling: touch)']: {
             maxHeight: windowHeight
           },
+          '@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none)': {
+            // For IE high contrast mode
+            borderLeft: `1px solid ${palette.neutralLight}`,
```
Comment (kkjeer) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: @betrue-final-final per your [comment](https://github.com/OfficeDev/office-ui-fabric-react/pull/7155#issuecomment-479558719) on #7155, should this border be any thicker?

Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: I was having the same thought from looking at the screenshot in the description  😄

File: packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts [hunk 2]
```diff
@@ -258,6 +258,11 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
           ['@supports (-webkit-overflow-scrolling: touch)']: {
             maxHeight: windowHeight
           },
+          '@media screen and (-ms-high-contrast: active), (-ms-high-contrast: none)': {
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: wait... I am confused on the part with `(-ms-high-contrast: none)`... does this mean the border will be both in HC and not in HC modes? And why we are not using the existing HC selector like here:
https://github.com/OfficeDev/office-ui-fabric-react/blob/7b108c2f0942dafa3193431ec631b66b433c2137/packages/office-ui-fabric-react/src/components/Breadcrumb/Breadcrumb.styles.ts#L67

Comment (kkjeer) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: Good question. I copied the change from this [commit](https://github.com/OfficeDev/office-ui-fabric-react/pull/7997/files#diff-f9b63ae6ae69f8d6ab55f2f3475b5a52L173). The border doesn't appear in high contrast mode.

Comment (Vitalius1) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: You should probably use the `[HighContrastSelector]` here as per [documentation](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/-ms-high-contrast#Values) the `none` is no longer supported. The `HighContrastSelector` is giving you the first part https://github.com/OfficeDev/office-ui-fabric-react/blob/7b108c2f0942dafa3193431ec631b66b433c2137/packages/styling/src/styles/CommonStyles.ts#L1

Comment (kkjeer) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: Updated to use `[HighContrastSelector`.

File: packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts (hunk 1 above)
Comment (dzearing) [reply packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts]: @betrue-final-final says 3px

**Pull request merged**